.gitignore

```diff
@@ -1,6 +1,6 @@
 composer.phar
 vendor/
-
+.idea*
```

Comment:
I think you are being overly critical. I will restate my original opinion, I don't see any harm having it here.